src/Sentry.Unity/UnityWebRequestTransport.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Sentry.Extensibility;
+using Sentry.Http;
+using Sentry.Protocol.Envelopes;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace Sentry.Unity
+{
+    internal class WebBackgroundWorker : IBackgroundWorker
+    {
+        private readonly SentryMonoBehaviour _behaviour;
+        private readonly UnityWebRequestTransport _transport;
+
+        public WebBackgroundWorker(SentryUnityOptions options, SentryMonoBehaviour behaviour)
+        {
+            _behaviour = behaviour;
+            _transport = new UnityWebRequestTransport(options);
+        }
+
+        public bool EnqueueEnvelope(Envelope envelope)
+        {
+            _ = _behaviour.StartCoroutine(_transport.SendEnvelopeAsync(envelope));
+            return true;
+        }
+
+        public Task FlushAsync(TimeSpan timeout) => Task.CompletedTask;
+
+        public int QueuedItems { get; }
+    }
+
+    internal class UnityWebRequestTransport : HttpTransportBase
```

Comment:
Even though it's not the goal of this PR. this transport has the potential to unblock usages outside WebGL. Mainly Console platforms. We can refactor things out to be reused later, so that perhaps a selected number of platforms. Possibly even: Anything that's not desktop and mobile

Reply:
that's why I've moved it to a separate file in the Sentry.Unity csproj - should be a good starting point without major git-diffs when we want to use for another platform.